addon/components/dialogs/form.js

```diff
@@ -44,17 +42,22 @@ export default FrostModalBinding.extend(PropTypesMixin, {
     assign(defaultProps, {
       animation: form,
       classModifier: 'form',
-      modal: 'frost-modal-dialog',
-      params: {
-        cancel: this.cancel,
-        confirm: this.confirm,
-        content: this.form,
-        links: this.links,
-        title: this.title
-      }
+      modal: 'frost-modal-dialog'
     })
 
     return defaultProps
-  }
+  },
+
+  // == Computed properties ===================================================
+
+  params: computed(function () {
+    return {
+      cancel: this.cancel,
+      confirm: this.confirm,
```

Comment:
how are you handling the confirm disabled case now? I see the 'hack' was removed.